source/server-discovery-and-monitoring/tests/rs/ls_timeout.json

```diff
@@ -45,30 +161,67 @@
           "b:27017": {
             "type": "RSSecondary",
             "setName": "rs"
+          },
+          "c:27017": {
+            "type": "Unknown"
+          },
+          "d:27017": {
+            "type": "RSGhost"
+          },
+          "e:27017": {
+            "type": "RSArbiter",
+            "setName": "rs"
           }
         },
         "topologyType": "ReplicaSetWithPrimary",
-        "logicalSessionTimeoutMinutes": 1,
+        "logicalSessionTimeoutMinutes": 2,
         "setName": "rs"
       }
     },
     {
       "responses": [
         [
-          "a:27017",
+          "c:27017",
           {
             "ok": 1,
-            "ismaster": true,
-            "hosts": [
-              "a:27017",
-              "b:27017"
-            ],
+            "ismaster": false,
             "setName": "rs",
+            "hidden": true,
             "logicalSessionTimeoutMinutes": 1,
             "minWireVersion": 0,
             "maxWireVersion": 6
           }
-        ],
+        ]
+      ],
+      "outcome": {
+        "servers": {
+          "a:27017": {
+            "type": "RSPrimary",
+            "setName": "rs"
+          },
+          "b:27017": {
+            "type": "RSSecondary",
+            "setName": "rs"
+          },
+          "c:27017": {
+            "type": "RSOther",
+            "setName": "rs"
+          },
+          "d:27017": {
+            "type": "RSGhost"
+          },
+          "e:27017": {
+            "type": "RSArbiter",
+            "setName": "rs"
+          }
+        },
+        "topologyType": "ReplicaSetWithPrimary",
+        "logicalSessionTimeoutMinutes": 1,
```

Comment:
This should be 2. I think we should ignore the RSOther. This specific RSOther is merely hidden, so its session timeout is ok, but in general SDAM says RSOther "servers may be hidden, starting up, or recovering." I'd prefer not to use their session timeout.

Reply:
Roger that.

Reply:
Done